Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
I agree that it's better to avoid dependencies, but is't that a copy of extension from `Hue`? It means if there is a bug/update we have to fix it twice because of code duplication. What do you think @hyperoslo/ios ?